content/tealpractices.md

```diff
@@ -0,0 +1,57 @@
+
+---
+layout: content
+name: Teal Practices
+---
+# Teal Practices
+
+* Daily Practices
+  * [Conflict resolution](https://reinventingorganizations-wiki.netlify.app/theory/conflict-resolution/)
```

Comment:
I would use the URL without the domain `/theory/conflict-resolution/` also here. (Same as in `tealconcepts.md`)

Reply:
Ah right, nice catch. Will do so through the CMS directly.